[2025-05-21T16:08:47.342+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-21T16:08:47.368+0000] {taskinstance.py:2631} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2023-03-13T00:00:00+00:00 [queued]>
[2025-05-21T16:08:47.380+0000] {taskinstance.py:2631} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2023-03-13T00:00:00+00:00 [queued]>
[2025-05-21T16:08:47.381+0000] {taskinstance.py:2884} INFO - Starting attempt 1 of 6
[2025-05-21T16:08:47.398+0000] {taskinstance.py:2907} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2023-03-13 00:00:00+00:00
[2025-05-21T16:08:47.408+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v02', 'get_name', 'scheduled__2023-03-13T00:00:00+00:00', '--job-id', '1512', '--raw', '--subdir', 'DAGS_FOLDER/dar_with_taskflow_api.py', '--cfg-path', '/tmp/tmpetk431l7']
[2025-05-21T16:08:47.411+0000] {standard_task_runner.py:105} INFO - Job 1512: Subtask get_name
[2025-05-21T16:08:47.409+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=4727) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-21T16:08:47.415+0000] {standard_task_runner.py:72} INFO - Started process 4738 to run task
[2025-05-21T16:08:47.489+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2023-03-13T00:00:00+00:00 [running]> on host 2032228d86df
[2025-05-21T16:08:47.601+0000] {taskinstance.py:3157} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v02' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2023-03-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-03-13T00:00:00+00:00'
[2025-05-21T16:08:47.602+0000] {taskinstance.py:740} INFO - ::endgroup::
[2025-05-21T16:08:47.605+0000] {python.py:240} INFO - Done. Returned value was: {'first_name': 'Jerry', 'last_name': 'Fridman'}
[2025-05-21T16:08:47.641+0000] {taskinstance.py:349} INFO - ::group::Post task execution logs
[2025-05-21T16:08:47.642+0000] {taskinstance.py:361} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v02, task_id=get_name, run_id=scheduled__2023-03-13T00:00:00+00:00, execution_date=20230313T000000, start_date=20250521T160847, end_date=20250521T160847
[2025-05-21T16:08:47.686+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-21T16:08:47.717+0000] {taskinstance.py:3924} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-21T16:08:47.720+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
